src/components/Footer.js

```diff
@@ -0,0 +1,29 @@
+import Vue from 'vue';
+
+export default function() {
+  return new Vue({
+    name: 'Footer',
+    el: ".footer",
+    template: require('src/templates/Footer.html'),
+    data: {
+      facebookUrl: 'https://www.facebook.com/sharer/sharer.php',
+      currentUrl: location.href
+    },
+    computed: {
+      shareUrl() {
+        return `${this.facebookUrl}?=${location.href}`
```

Comment:
I think the reason the `shareUrl` in the footer is not updating when the hash changes is here: if this instead does `${currentUrl}` then Vue will know to update the DOM when the `currentUrl` changes. Tested it and seems to work!